addons/stock/tests/test_move2.py

```diff
@@ -511,6 +511,68 @@ def test_pick_pack_ship_return(self):
         self.assertEqual(set(return_pick_picking.move_lines.move_orig_ids.ids), set((picking_pick.move_lines | return_pack_picking.move_lines).ids))
         self.assertEqual(len(return_pick_picking.move_lines.move_dest_ids), 0)
 
+    def test_put_in_pack(self):
```

Comment:
""" In a pick pack ship scenario, create two packs in pick and check that
they are correctly recognised and handled by the pack and ship picking.
Along this test, we'll use `action_toggle_processed` to process a pack
from the `entire_package_ids` one2many and we'll directly fill the move
lines, the latter is the behavior when the user did not enable the display
of entire packs on the picking type. 
""'